src/EnsembleKalmanProcess.jl

```diff
@@ -336,4 +337,324 @@ function update_ensemble!(ekp::EnsembleKalmanProcess{FT, IT, Sampler{FT}}, g_in:
 end
 
 
+
+
+mutable struct  Unscented{FT<:AbstractFloat, IT<:Int} <: Process
+    "a vector of arrays of size N_parameters containing the mean of the parameters (in each uki iteration a new array of mean is added)"
+    u_mean::Vector{Array{FT, 1}}
+    "a vector of arrays of size (N_parameters x N_parameters) containing the covariance of the parameters (in each uki iteration a new array of cov is added)"
+    uu_cov::Vector{Array{FT, 2}}
+    "a vector of arrays of size N_y containing the predicted observation (in each uki iteration a new array of predicted observation is added)"
+    obs_pred::Vector{Array{FT, 1}}
+    "weights in UKI"
+    c_weights::Array{FT, 1}
+    mean_weights::Array{FT, 1}
+    cov_weights::Array{FT, 1}
+    "covariance of the artificial evolution error"
+    Σ_ω::Array{FT, 2}
+    "covariance of the artificial observation error"
+    Σ_ν::Array{FT, 2}
+    "regularization parameter"
+    α_reg::FT
+    "regularization vector"
+    r::Array{FT, 1}
+    "update frequency"
+    update_freq::IT
+    "current iteration number"
+    iter::IT
+end
+
+# outer constructors
+function EnsembleKalmanProcess(
+    obs_mean::Array{FT, 1},
+    obs_noise_cov::Array{FT, 2},
+    process::Unscented{FT, IT};
+    Δt=FT(1)) where {FT<:AbstractFloat, IT<:Int}
+
+#initial parameters stored as columns
+init_params=[]
```

Comment:
Here we want to create the initial sigma ensemble from the process. we can define in this function
```Julia
init_params = update_ensemble_prediction!(process)
```

Reply:
updated! Thanks!

Reply:
updated, thanks!